schema.sql

```diff
@@ -1,5 +1,27 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id serial primary key,
+	name varchar(50),
+	date_of_birth date,
+	escape_attempts INT,
+	neutered boolean,
+	weight_kg decimal
 );
+
+-- add species by updating table animal
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(50);
```

Comment:
- [ Required ] I think you didn't understand what the previous reviewer meant, if you check the project requirements carefully for this `milestone` you'll notice that the following  `SQL` statements are missing in your `schema.sql` file.
- Modify `Animals` table
   - SQL statement to `remove` column `species`
   - SQL statement to `add` column `species_id` which is a foreign key referencing `species` table
   - SQL statement `add` column `owner_id` which is a foreign key referencing the `owners` table

## Project requirements
![Screenshot from 2023-04-01 13-50-23](https://user-images.githubusercontent.com/16392046/229284920-5fb14d16-9495-4dee-acf2-84cba26e3f3a.png)